Sensus.Shared/SensusServiceHelper.cs

```diff
@@ -738,7 +738,7 @@ public void AddScriptToRun(Script script, RunMode runMode)
 
                 if (runMode == RunMode.Multiple)
                 {
-                    _scriptsToRun.Insert(0, script);
+                    _scriptsToRun.Insert(ScriptToAddIndex(script), script);
                     modifiedScriptsToRun = true;
                 }
```

Comment:
Think about line 748 below. Do we still need it? If so, is the sort correct?

Reply:
You are correct, that line is no longer needed.